### ISSUE TITLE
Has the App Runner service been defunded?

### ISSUE THREAD

**Issue by rhbecker** [aws/apprunner-roadmap#188]
In the first `~2.5` months of 2023 (`01/06` - `03/16`) ...

- `14` cards moved to `Just Shipped` status.
- `15` other card status changes occurred.

In the subsequent `~2` months (`03/17` - today, `05/19`) ...

- `1` card moved to `Just Shipped` status.
- No other card status changes occurred.

Anecdotally, it also _seems_ like there's been less engagement from the team on posted issues / issue updates.

I understand this isn't the sort of question that I probably should be posting as an issue on this repo, but as someone assessing whether to guide an organization towards use of this service, the reduced activity is concerning, and some transparency around the reasons would be useful and appreciated.

**Comment by janaka** [comment]
Great question @rhbecker. Let's hope we get an answer to this at least.

**Comment by jsheld** [comment]
No, it has not been defunded. Of recent, the team just introduced additional vCPU and memory options. See the doc post here: https://aws.amazon.com/about-aws/whats-new/2023/04/aws-app-runner-compute-configurations/. This released on 4/5/2023. Release notes for other items can be found here: https://docs.aws.amazon.com/apprunner/latest/relnotes/relnotes.html. The team is actively working on additional items and updates to the roadmap are coming.

**Comment by rhbecker** [comment]
That's a relief to hear, and I really appreciate the transparency - both with respect to your response, and this repo in general.

If I didn't see app runner as a very promising service, I wouldn't care enough to ask, so please accept these sorts of inquiries as a sign of support!

**Comment by jsheld** [comment]
No worries. Ultimately, the roadmap will have items that are added and transition from one status to another. You can follow these projects through that pipeline until they are ultimately shipped. Please be advised that not everything we are working on may show up there, as the element of surprise is the best way to delight customers and keep AWS competitive. We appreciate your feedback and will resolve this issue accordingly. If there is something else that comes up, feel free to reach out.
